api/resources/views/posts/edit.blade.php

```diff
@@ -29,11 +29,11 @@
                         {{Form::textarea('content', null, ["class" => 'form-control'])}}
 
                         {{Form::label('job_form', "Attach the Job Form:", ["class" => 'h5 mt-3'])}}
-                        {{Form::select('job_form', array(null => null, 'Job1' => 'Job1', 'Job2' => 'Job2'), 0,  array('class' =>'form-control'))}}
+                        {{Form::select('job_form', ['' => ''] + $forms->pluck('name', 'id')->toArray(), $post->form_id,  array('class' =>'form-control'))}}
                         
                         <div>
                             {{Form::label('academic_group', "Add the Interest Group:", ["class" => 'h5 mt-3'])}}
```

Comment:
Is anyone supposed to be allowed to add the interest group? Why is it only available on the edit page and not on the create page?

Reply:
Well it's under our created requirements xD That ONLY interest group can be added by the lecturer. But I haven't implemented it yet